test/suites/common-container-chains/test-pallet-xcm-executor-utils/test-custom-policy.ts

```diff
@@ -58,13 +58,8 @@ describeSuite({
                         {
                             allowedAssets: [
                                 {
-                                    concrete: {
-                                        parents: 2,
-                                        interior: { Here: null },
-                                    },
```

Comment:
Why fungible assets were removed from allowed assets?

Reply:
I think it is because now they are only identified by their Location

Reply:
Actually I think this test was wrong before, I am not sure why it was working. According to my knowledge the right type is a multilocation. @fgamundi do you know why we put a fungible there¿?

Reply:
Not sure why fungible was there. The right type is indeed a `MultiLocation` (`AssetId`, really). Must have gotten confused with the actual `XcmFragment`s in the test.
I just did some tests and seems like Polkadot.js just ignores any additional data that doesn't belong to the correct type, ignoring `fun` in this case